camera_name: g_bot_camera
robot_name: g_bot
eye_on_hand: false
robot_base_frame: workspace_center
robot_effector_frame: g_bot_flange
robot_effector_tip_frame: g_bot_gripper_tip_link
camera_parent_frame: workspace_center
marker_map: aruco-26-70x70-5
enclosed_marker: false
planarity_tolerance: 0.003
initpose: [0.00, -0.40, 0.30, 90, 90, 0]
keyposes:
  - [-0.12, -0.18, 0.17, 30, 150, -90]
  - [-0.12, -0.05, 0.17, 30, 150, -90]

  - [ 0.00, -0.05, 0.17, 30, 150, -90]
  - [ 0.00, -0.18, 0.17, 30, 150, -90]

  - [ 0.12, -0.18, 0.17, 30, 150, -90]
  - [ 0.12, -0.05, 0.17, 30, 150, -90]
